src/libraries/System.Security.Cryptography.Xml/tests/EncryptionMethodTest.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Security.Cryptography.Xml;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class EncryptionMethodTest
+    {
+        [Fact]
+        public void GetXml_ReturnsExpectedXml()
+        {
+            var encryptionMethod = new EncryptionMethod("http://www.w3.org/2001/04/xmlenc#aes256-cbc");
+            var xmlElement = encryptionMethod.GetXml();
```

Comment:
This use of `var` violates our style guidelines.

Never using `var` is always style-conforming in this repository.  If you think that a line like `EncryptionMethod encryptionMethod = new EncryptionMethod(...);` is too long, `var` is acceptable there (the type name is present on the line), but target-typed-new is probably the better choice over var (`EncryptionMethod encryptionMethod = new(...);`).

Which means that for new code, `var` would only really be used in a cast statement, if ever.